### ISSUE TITLE
Does anyone find the public signaling server performing worse and worse?

### ISSUE THREAD

**Issue by lucastwong** [peers/peerjs#747]
I'm at the point where I can get anything to work (on the demo page or my app) especially around the weekend. Does anyone notice that it's been worse and worse every single week?

**Comment by Cheesetouched** [comment]
You're absolutely correct. I am facing issues as well. I check the network logs yesterday, and it said 'Maximum concurrent user limit reached'. My demos are not working at all.

**Comment by lucastwong** [comment]
I have made it work on AWS.

**Comment by morgan3d** [comment]
I'm continuously hitting this. It would be helpful to hear from the maintainers if there's something we can do to increase the availability of the public PeerJS cloud. For example, is there a donation level at which they'd be able to scale up the public server?
